vyper/venom/effects.py

```diff
@@ -0,0 +1,59 @@
+from enum import Flag, auto
+
+
+class Effects(Flag):
```

Comment:
modeling code (e.g., for code copy) isn't useful?

Reply:
It's not that useful, since it can't change during execution

Reply:
(Ay least that I know of)